doc/classes/EditorImportPlugin.xml

```diff
@@ -227,5 +227,15 @@
 				This method must be overridden to do the actual importing work. See this class' description for an example of overriding this method.
 			</description>
 		</method>
+		<method name="append_import_external_resource">
+			<return type="int" enum="Error" />
+			<param index="0" name="path" type="String" />
+			<param index="1" name="custom_options" type="Dictionary" default="{}" />
+			<param index="2" name="custom_importer" type="String" default="&quot;&quot;" />
+			<param index="3" name="gen_params" type="Variant" default="null" />
```

Comment:
```suggestion
			<param index="3" name="generate_parameters" type="Variant" default="null" />
```

Godot tends to trend towards fully spelt out names in core apis.

Reply:
if we do this, I think it should say generated, not generate.

This needs to match the name in the .import file, since users of this API will need to read the remap section to retrieve this data afterwards... so we would need to change the underlying `_reimport_file` code too

Reply:
Updated.

Reply:
We clobbered each other's pushes. but `generator` is fine.